index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"> <a><img src="images/image_geometry_1.png" alt="logo image"></a></div>
+        <nav>
+            <div id="toolbar">≡</div>
+        </nav>
+    </header>
+    <section class="hello">
+        <div hello-text>
+          <header>
+            <h1 class="name">Javed Shahbaz</h1>
+          </header>
```

Comment:
- [ ]  Good job using the `hx` :clap: , however  for semantic reason , you should remove the `header` , :smile: